ztests/suite/format/hello.azng

```diff
@@ -0,0 +1 @@
+�a	b	�a	bhelloworldgoodnight�
```

Comment:
Can you mark this file as binary for git? It didn't seem to detect it automatically.

Reply:
Leaving this "as is" as we agreed...